scanner/src/main/kotlin/scanners/BoyterLc.kt

```diff
@@ -87,28 +98,36 @@ object BoyterLc : LocalScanner() {
         }
     }
 
+    override fun getConfiguration() = configuration.joinToString(" ")
+
     override fun getVersion(executable: String) =
             getCommandVersion(scannerPath.absolutePath, transform = {
                 // "lc --version" returns a string like "licensechecker version 1.1.1", so simply remove the prefix.
                 it.substringAfter("licensechecker version ")
             })
 
-    override fun scanPath(path: File, resultsFile: File): Result {
+    override fun scanPath(path: File, resultsFile: File, provenance: Provenance, specification: ScannerSpecification)
+            : ScanResult {
+        val startTime = Instant.now()
```

Comment:
Can we move the repeating code for `startTime` / `endTime` to `LocalScanner`?

Reply:
This is difficult because the method is supposed to return a `ScanResult` which already contains the timestamps in its summary.